File: Python/Projects/calculator.py
print("                  :Calculator:                     ")
print("===================================================")
print("""
1.Addition  
2.Subtraction  
3.Multiplication  
4.Division   
5.Remainder  
""")

option = input("Enter Option: ")
option = int(option) #Converting option into an integer to match with the if statements.


def addition():
    num = input("Enter Number: ")
    num1 = input("Enter Second Number: ")
    sum = num + num1
    print("Sum = ",sum)

def subtraction():
    num = input("Enter Number: ")
    num1 = input("Enter Second Number: ")
    sub = num - num1
    print("Subtraction = ",sub)

def multiplication():
    num = input("Enter Number: ")
    num1 = input("Enter Second Number: ")
    prod = num * num1
    print("Product = ",prod)

def remainder():
    num = input("Enter Number: ")
    num1 = input("Enter Second Number: ")
    rem = num % num1
    print("Remainder = ",rem)

def division():
    num = input("Enter Number: ")
    num1 = input("Enter Second Number: ")
    div = num/num1
    print("Division = ",div)


if option == 1:
    addition()
elif option == 2:
    subtraction()
elif option == 3:
    multiplication()
elif option == 4:
    division()
elif  option == 5:
    remainder()


print("===================================================")

File: C++/Projects/Calculator.cpp
#include <iostream>
using namespace std;

void addition(){
    int x;
    int y;
    cout << "Enter Number: ";
    cin >> x;
    cout << "Enter Second Number: ";
    cin >> y;
    int sum = x + y;
    cout << "Sum = " << sum << endl;
}

void subtraction(){
    int x;
    int y;
    cout << "Enter Number: ";
    cin >> x;
    cout << "Enter Second Number: ";
    cin >> y;
    int sub = x - y;
    cout << "Subtraction = " << sub << endl;
}

void multiplication(){
    int x;
    int y;
    cout << ""
    cout << "Enter Number: ";
    cin >> x;
    cout << "Enter Second Number: ";
    cin >> y;
    int prod = x * y;
    cout << "Product = " << prod << endl;
}

void division(){
    int x;
    int y;
    cout << "Enter Number: ";
    cin >> x;
    cout << "Enter Second Number: ";
    cin >> y;
    int div = x / y;
    cout << "Division = " << div << endl;
}

void remainder(){
    int x;
    int y;
    cout << "Enter Number: ";
    cin >> x;
    cout << "Enter Second Number: ";
    cin >> y;
    int rem = x % y;
    cout << "Remainder = " << rem << endl;
}

int main(){
    int opt;
    cout << "Enter Option: ";
    cin >> opt;
    if (opt == 1){
        addition();
    }else if (opt == 2) {
        subtraction();
    }else if (opt == 3){
        multiplication();
    }else if (opt == 4){
        division();
    }else if (opt == 5){
        remainder();
    }
}
